### ISSUE TITLE
Help with filters

### ISSUE THREAD

**Issue by adamehunt** [pistacchio/obsidian-enhanced-tables#6]
Hello,

This is a cool plugin! I am interested in using it for a video game spreadsheet — and I like that each entry doesn't have to be a separate note in a database.

I am having a few issues with the filtering system.

1. I would like to be able to filter by the table header "co-op", using the values "Yes" or "No"
2. I would also like to be able to filter by the table header "platform" using the values "Arcade", "Dreamcast" and "Steam" in the example below
3. I am having trouble getting the "searchable" filter to work. I wouldn't mind searching by all of the headers, but "game" and "platform" are the most important.

Thank you. I appreciate your help!

Here is my example code.

```yaml enhanced-tables

columns:
  Game:
    nowrap: true
    searchable: true
  interest:
    type: enum
    enum:
      '1': '⭐️'
      '2': '⭐️⭐️'
      '3': '⭐️⭐️⭐️'
filters:
 High Interest: Number($row.interest) > 2
 Multiplayer: Number($row.players) > 1
 
pagination:
  page-size: 5
  page-sizes:
   - 5
   - 10
   - 20
style: |
   th {
     background-color: var(--color-base-50) !important;
     color: var(--color-base-70) !important;
   }
# hide-controls: true
hide-configuration: true
```

| game                                             | platform  | status   | co-op | players | interest |
| :----------------------------------------------- | :-------- | :------- | :---- | :------ | :------- |
| Dead Connection                                  | Arcade    | Unplayed | No    | 1       | 1        |
| Gain Ground                                      | Arcade    | Unplayed |       |         | 3        |
| Gun force 2 / Geoforce                           | Arcade    | Unplayed |       |         |          |
| Moonwalker                                       | Arcade    | Finished | Yes   |         |          |
| Ninja Baseball Bat Man                           | Arcade    | Unplayed | Yes   |         | 2        |
| The Outfoxes                                     | Arcade    | Unplayed |       |         |          |
| Border Down                                      | Dreamcast | Unplayed |       |         |          |
| Bust a Move 4                                    | Dreamcast | Unplayed |       |         | 3        |
| Chu Chu Rocket                                   | Dreamcast | Unplayed |       |         | 3        |
| Dynamite Cop — co-op                             | Dreamcast | Unplayed |       |         | 2        |
| Columns                                          | Dreamcast | Unplayed |       |         | 1        |
| Cyber Troopers Virtual On                        | Dreamcast | Unplayed |       |         | 1        |
| Guardian Heroes                                  | Dreamcast | Unplayed |       |         | 3        |
| House of the Dead 2                              | Dreamcast | Played   |       |         | 2        |
| Ikaruga                                          | Dreamcast | Unplayed | Yes   | 2       | 2        |
| Teenage Mutant Ninja Turtles: Shredder's Revenge | Steam     | Unplayed | Yes   |         | 3        |

**Comment by pistacchio** [comment]
Hi!
Thanks for trying out the plugin!

So, the first isse I see in your configuration is that in the table the first column is called `game` (lowercase "g") while in the cofiguration you refer to it as `Game` (uppercase "G"). The cases must match.

Your example helped me find out (and fix) a bug in the search funcionality. I've just uploaded the version with the fix (`1.0.5`). Searching for "game" and "platform" should now work if you update the version of the plugin and set both columns as `searchable: true`.

That said, try out the following configuration!

```yaml enhanced-tables
columns:
  game:
    nowrap: true
    searchable: true
  platform:
    searchable: true
  interest:
    type: enum
    enum:
      '1': '⭐️'
      '2': '⭐️⭐️'
      '3': '⭐️⭐️⭐️'
  co-op:
    alias: coop
filters:
 High Interest: Number($row.interest) > 2
 Multiplayer: Number($row.players) > 1
 Co-op: $row.coop === 'Yes'
 No Co-op: $row.coop !== 'Yes'
 Arcade: $row.platform === 'Arcade'
 Dreamcast: $row.platform === 'Dreamcast'
 Steam: $row.platform === 'Steam'
 
pagination:
  page-size: 5
  page-sizes:
   - 5
   - 10
   - 20
style: |
   th {
     background-color: var(--color-base-50) !important;
     color: var(--color-base-70) !important;
   }
# hide-controls: true
hide-configuration: true
```

**Comment by adamehunt** [comment]
That worked great, thank you!
